scripts/build-glean-metadata.py

```diff
@@ -141,16 +153,19 @@ def get_app_variant_description(app):
         continue
 
     if not app_groups.get(app.app_name):
-        app_groups[app.app_name] = {
-            "app_name": app.app_name,
-            "app_description": app.app["app_description"],
-            "canonical_app_name": app.app["canonical_app_name"],
-            "deprecated": app.app.get("deprecated", False),
-            "url": app.app["url"],
-            "notification_emails": app.app["notification_emails"],
-            "app_ids": [],
-            "annotation": (annotations_index.get(app.app_name, {}).get("app")),
-        }
+        app_groups[app.app_name] = _incorporate_annotation(
```

Comment:
We don't actually use tags for applications yet, but it doesn't hurt to add them.

Reply:
Actually this doesn't make sense, since applications do have a tag definition. Left a comment to this effect in the source, we probably should clear this up at some point